From dc45caf800090d531c60076c92a134b1e6b54885 Mon Sep 17 00:00:00 2001
From: psathisx <pinayourx.sathish@intel.com>
Date: Fri, 22 Feb 2013 19:30:02 +0530
Subject: [PORT FROM MAIN] Revert "[WORKAROUND][Audio-MW]: While trying to play WAV(PCM) 192kHz song (which is not"

BZ: 68252

Reverting this patch since a generic fix which will cover this issue for
all the platform supported audio-formats has been provided with the patch
http://android.intel.com/89798 which is merged.

This reverts commit 1c6f770178cc1fe2ec70f196d44d1875de17ecd0.

Change-Id: I443933299e4a35203e0432fd9f60e13150c7870d
Orig-Change-Id: I7776b7cf5034c509feb37ede676ad92425dd30cb
Reviewed-on: http://android.intel.com:8080/93349
Reviewed-by: Sathish, PinayourX <pinayourx.sathish@intel.com>
Reviewed-by: Kandasamy, Muthukumar <muthukumar.kandasamy@intel.com>
Reviewed-by: Sameullah, MazharX <mazharx.sameullah@intel.com>
Reviewed-by: Krishna, Samaga <samaga.krishna@intel.com>
Tested-by: Gupta, ArvindX K <arvindx.k.gupta@intel.com>
Reviewed-by: P C, SreekanthX <sreekanthx.p.c@intel.com>
Reviewed-by: buildbot <buildbot@intel.com>
Tested-by: buildbot <buildbot@intel.com>
---
 media/libstagefright/WAVExtractor.cpp |    6 +-----
 1 files changed, 1 insertions(+), 5 deletions(-)

diff --git a/media/libstagefright/WAVExtractor.cpp b/media/libstagefright/WAVExtractor.cpp
index 353686a..22af6fb 100644
--- a/media/libstagefright/WAVExtractor.cpp
+++ b/media/libstagefright/WAVExtractor.cpp
@@ -32,10 +32,6 @@
 
 #define CHANNEL_MASK_USE_CHANNEL_ORDER 0
 
-//Maximum input sampling rate that can be handled by the resampler.
-//This is a limitation of the resampler and not the WAV format.
-#define MAX_INPUT_SAMPLING_RATE 96000
-
 namespace android {
 
 enum {
@@ -210,7 +206,7 @@ status_t WAVExtractor::init() {
 
             mSampleRate = U32_LE_AT(&formatSpec[4]);
 
-            if (mSampleRate <= 0 || mSampleRate > MAX_INPUT_SAMPLING_RATE) {
+            if (mSampleRate == 0) {
                 return ERROR_MALFORMED;
             }
 
-- 
1.7.4.1

